### DISCUSSION TITLE
Is there any chance I may have a call with you on Discord or the like?

### DISCUSSION THREAD

**Discussion by CodingMadness** [dotnet/dotNext#160] [Ideas]
Title.

I am working on something, high performant related and I think it can be quite useful but I really would like your opinion on a method, (the rest of the code is nearly done!) 

I would be ultra happy if we could arrange that ^^

Best regards!

**Comment by sakno** [comment]
> our opinion on a method

Is this some top secret code? It's better to start conversation here or in LinkedIn for me, not sure about available time slots on this week.

**Reply by CodingMadness** [reply]
Sry for late reply, forgot to answer... I think it COULD be something of use for your "DotNext" framework, so I actually intented to discuss what i have done there (should not take more than 5-10 min) and explain why it could be something for DotNext to be using. 

Would be glad to heal from you :--)

**Reply by sakno** [reply]
.NEXT project has its own chat in Gitter. It's better to start from there. You can find a link to it on the badge in README.
